### DISCUSSION TITLE
Sorry i am just noob...

### DISCUSSION THREAD

**Discussion by SamuelPalubaCZ** [toltec-dev/toltec#848] [Support] [answered]
i cannot find what version should i downgrade for rm1 :(((

**Comment by Eeems** [comment, marked as answer]
The toltec website indicates the latest supported OS version number.
For information on how to downgrade: https://remarkable.guide/faqs.html#can-i-downgrade-to-a-different-os-version

**Reply by kubycsolutions** [reply]
Note that the usually recommended downgrade tool actually lets you specify "toltec" as a version, and will switch to the most recent version that Toltec supports.

Remember to turn off auto-update immediately after downgrading, or you may get auto-upgraded again before you can install Toltec.
